networking/eth2/src/main/java/tech/pegasys/teku/networking/eth2/gossip/forks/versions/GossipForkSubscriptionsPhase0.java

```diff
@@ -219,6 +239,12 @@ public void publishBlock(final SignedBeaconBlock block) {
     blockGossipManager.publishBlock(block);
   }
 
+  @Override
+  public void publishBlockAndBlobsSidecar(
+      final SignedBeaconBlockAndBlobsSidecar blockAndBlobsSidecar) {
+    // Does not apply to this fork.
```

Comment:
I'd maybe go with interface defaults for this and 2 others (`publishSyncCommitteeMessage`, `publishSyncCommitteeContribution`) to not leak them to `Phase0` implementation but definitely up to you